teacher.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, name, specialization)
+    super(age, name)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+  private :can_use_services?
```

Comment:
- [ ] [OPTIONAL] In my opinion these are not important 👍🏼  [` private :can_use_services?`]